federation/pkg/federation-controller/util/cluster_util.go

```diff
@@ -146,3 +149,58 @@ func GetClientsetForCluster(cluster *federation_v1beta1.Cluster) (*fedclientset.
 	}
 	return nil, err
 }
+
+// Returns a bool to indicate if the object should be forwarded to a cluster
+func SendToCluster(clusterLabels map[string]string, annotations map[string]string) (bool, error) {
```

Comment:
How about adding a util/cluster_selector package and putting this there?
@tsandal's code might also want to reuse some util funcs and we want to minimise code in generic util package.